#!/bin/sh -e
## 02_makefile.dpatch.dpatch by Guido Trotter <ultrotter@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Modify Makefile.am not to install documents: we will do in debian/rules

if [ $# -ne 1 ]; then
    echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
    exit 1
fi

[ -f debian/patches/00patch-opts ] && . debian/patches/00patch-opts
patch_opts="${patch_opts:--f --no-backup-if-mismatch}"

case "$1" in
       -patch) patch $patch_opts -p1 < $0;;
       -unpatch) patch $patch_opts -p1 -R < $0;;
        *)
                echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
                exit 1;;
esac

exit 0

@DPATCH@
diff -urNad /home/ultrotter/docs/projects/deb/freej-0.5/Makefile.am freej-0.5/Makefile.am
--- /home/ultrotter/docs/projects/deb/freej-0.5/Makefile.am	2003-06-04 16:05:30.000000000 +0200
+++ freej-0.5/Makefile.am	2003-06-04 16:06:05.000000000 +0200
@@ -1,4 +1,4 @@
-docs_DATA = README COPYING USAGE AUTHORS NEWS
-docsdir = ${prefix}/doc/${PACKAGE}-${VERSION}
+#docs_DATA = README COPYING USAGE AUTHORS NEWS
+#docsdir = ${prefix}/doc/${PACKAGE}-${VERSION}
 
 SUBDIRS = src filters
